samples/nrf9160/lwm2m_client/prj.conf

```diff
@@ -11,28 +11,60 @@ CONFIG_NET_IPV4=y
 CONFIG_NET_SOCKETS=y
 CONFIG_NET_SOCKETS_OFFLOAD=y
 
-# LwM2M
-CONFIG_LWM2M_CLIENT_UTILS=y
-CONFIG_LWM2M_CLIENT_UTILS_LOG_LEVEL_DBG=y
+# These two settings are needed due to a error in zephyrs lwm2m implementation.
+# The error occurs when a multi instance resource is set to have a maximum of one instance.
+# When the fix for this error is integrated, which it presumably will be in NCS 1.7,
+# these settings can be safely removed. See zephyr PR #36657 for details.
+CONFIG_NET_IF_MAX_IPV4_COUNT=2
+CONFIG_LWM2M_CONN_MON_APN_MAX=2
+
+# Sensors
+CONFIG_I2C=y
+CONFIG_SPI=y
+CONFIG_SENSOR=y
+
+# UI
+CONFIG_UI_LOG_LEVEL_DBG=y
+
+# LwM2M and IPSO
 CONFIG_LWM2M=y
+CONFIG_LWM2M_COAP_BLOCK_SIZE=512
+CONFIG_LWM2M_ENGINE_MAX_OBSERVER=15
+CONFIG_LWM2M_ENGINE_MAX_MESSAGES=15
+CONFIG_LWM2M_ENGINE_MAX_PENDING=15
+CONFIG_LWM2M_ENGINE_MAX_REPLIES=15
+CONFIG_LWM2M_ENGINE_DEFAULT_LIFETIME=60
 CONFIG_LWM2M_DNS_SUPPORT=y
 CONFIG_LWM2M_RW_JSON_SUPPORT=n
 CONFIG_LWM2M_SERVER_DEFAULT_PMIN=1
 CONFIG_LWM2M_SERVER_DEFAULT_PMAX=300
+CONFIG_LWM2M_CLIENT_UTILS=y
+CONFIG_LWM2M_CLIENT_UTILS_LOCATION_OBJ_SUPPORT=n
```

Comment:
The implementation in the LwM2M utils library is rather unfinished, so I decided it would probably be easier to make my own. I do agree that we should rather fix the library, but I also think that the official implementation will probably require a lot more polish than what we have made here.

Reply:
@VeijoPesonen what's your say here? I guess we could update the utils lib at a later time? The PR is already pretty large, getting the utils lib update involved at this point might be a bit problematic.

Reply:
> 
> 
> @VeijoPesonen what's your say here? I guess we could update the utils lib at a later time? The PR is already pretty large, getting the utils lib update involved at this point might be a bit problematic.

Yeah, no need to solve all the issues at one time. It would be good to write down a TODO list somewhere. The PR description could be one place to put it so it wouldn't get lost.